tests/test_service.py

```diff
@@ -298,4 +306,6 @@ def test_create_ppr_db_error(
             ppr_data, common_context, deployment,
             dbsession, connect_client, logger,
         )
-    assert ex.value.message == "Database error occurred."
+    assert ex.value.message == (
```

Comment:
I think better to come up with another test for DB error, because now it is duplicated with another one that file already exists.